ia-wishlist-bot/adding_wishlist_ol.py

```diff
@@ -0,0 +1,104 @@
+import sys
+from olclient.openlibrary import OpenLibrary
+import olclient.common as common
+
+import csv, ast
+import requests
+import json
+import unittest
+
+ol = OpenLibrary()
+FILE = 'ia-data/new_wishlist_salman_1000.csv'
+
+class TestWishlistAddBook(unittest.TestCase):
+    def test_parse_wishlist_csv_row_to_dict(self):
+        csv = ["Larks in a paradise: New Zealand portraits", "['McNeish, James', 'Friedlander, Marti']", "eng", "1974", "16289249", "0002114976", "9780002114974"]
+        # expected = {"title": "foo", "authors": ['bar'], "language": "baz", "date": }
+        expected = {"title": "Larks in a paradise: New Zealand portraits", "authors": [
+            'McNeish, James', 'Friedlander, Marti'], "language": "eng", "date": "1974", "oclc": "16289249", "isbn10": "0002114976", "isbn13": "9780002114974"}
+
+        self.assertTrue(parse_wishlist_csv_row_to_dict(csv) == expected)
+
+    def test_check_if_author_exists(self):
+        author = {'author_name': 'Dan Brown',
```

Comment:
Maybe we should just use author_name for here (until the other fields are available)